src/mlte/store/underlying/fs.py

```diff
@@ -0,0 +1,458 @@
+"""
+mlte/store/underlying/fs.py
+
+Implementation of local file system artifact store.
+"""
+
+
+import json
+from typing import Any
+from pathlib import Path
+from mlte.store.store import Store, StoreSession, StoreURI
+from mlte.context.model import (
+    Namespace,
+    Model,
+    Version,
+    NamespaceCreate,
+    ModelCreate,
+    VersionCreate,
+)
+from mlte.artifact.model import ArtifactModel
+from mlte.store.query import Query
+
+import mlte.store.error as errors
+
+
+def _parse_root_path(uristr: str) -> Path:
+    """
+    Parse the root path for the backend from the URI.
+    :param uri: The URI
+    :type uri: str
+    :return: The parsed path
+    :rtype: Path
+    """
+    assert uristr.startswith("fs://") or uristr.startswith(
+        "local://"
+    ), "Broken precondition."
+    prefix = "fs://" if uristr.startswith("fs://") else "local://"
+    return Path(uristr[len(prefix) :])
+
+
+def _rmtree(root: Path):
```

Comment:
I believe we can. Since we were using the pathlib library for everything (which, in the past, I have not used that frequently), I was trying to look for an implementaiton that would only use Pathlib as well. But if you don't have a preference, I can change to shutil.rmtree()